integration_tests/src/main/python/spark_session.py

```diff
@@ -94,5 +94,6 @@ def with_gpu_session(func, conf={}):
     copy['spark.rapids.sql.decimalType.enabled'] = 'true'
     return with_spark_session(func, conf=copy)
 
+# databricks runtime 8.x and lower show 3.1.0 even though really 3.1.1
 def is_before_spark_311():
-    return spark_version() < "3.1.1"
+    return (is_databricks_runtime() and spark_version() < "3.1.0") or (not is_databricks_runtime() and spark_version() < "3.1.1")
```

Comment:
I committed this without thinking about it, this won't work because databricks is < 3.1.1 so it matches the first one which we don't want because we want the 310db to show as not before 311, I'm going revert

Reply:
Note tests fail on databricks 310db which is why I caught this